sktime/forecasting/naive.py

```diff
@@ -60,14 +73,22 @@ class NaiveForecaster(_BaseWindowForecaster):
     >>> y_pred = forecaster.predict(fh=[1,2,3])
     """
 
-    _tags = {"requires-fh-in-fit": False}
+    _tags = {
+        "requires-fh-in-fit": False,
+        "handles-missing-data": False,  # todo: switch to True if GH1367 is fixed
+    }
 
     def __init__(self, strategy="last", window_length=None, sp=1):
         super(NaiveForecaster, self).__init__()
         self.strategy = strategy
         self.sp = sp
         self.window_length = window_length
 
+        # Override tag for handling missing data
+        # todo: remove if GH1367 is fixed
+        if self.strategy in ("last", "mean"):
+            self.set_tags(**{"handles-missing-data": True})
```

Comment:
you could also write, perhaps more easy to read: `self.set_tags(handles-missing-data=True)`

Reply:
I don't think that is possible since the name would be interpreted with two minus symbols.

Reply:
oh, ouch - we should have thought of this in naming tags.
In fact, tags strings should be valid variable names.